.gitignore

```diff
@@ -0,0 +1,26 @@
+Makefile
+Makefile.in
+/ABOUT-NLS
+/INSTALL
+/aclocal.m4
+/autom4te.cache/
+/compile
+/config.*
+/configure
+/install-sh
+/m4/
+/missing
+/mate-wayland-session-*.tar.*
+/omf.make
+/xmldocs.make
+po/*.gmo
+po/*.sed
+po/*.sin
+po/*.header
+po/Rules-quot
+po/Makevars.template
+po/Makefile.in.in
+po/POTFILES
+po/mate-wayland-session.pot
+po/stamp-po
+session/MATE.desktop
```

Comment:
Mybe we want to use [git.mk](https://raw.githubusercontent.com/behdad/git.mk/) here as we do in other repos, in which case the commit adding this `.gitignore` can be removed from this PR.

Reply:
I have zero familiarity with using that, will go with .gitignore for the moment but look at that for potential later use